<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QWidget" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>783</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>645</width>
    <height>783</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>645</width>
    <height>783</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>833</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>833</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>682</width>
     <height>801</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>46</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>8</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollBar:vertical{
	width:0px;
}
</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>2000</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>681</width>
      <height>2000</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>681</width>
      <height>5000</height>
     </size>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000;</string>
    </property>
    <widget class="QLabel" name="title">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 255, 255)
</string>
     </property>
     <property name="text">
      <string>Дневник снов</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_add">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: black;
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255))
</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_what">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000;
background-color: rgb(255, 255, 255);
border-radius: 10px;




</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteAll_button">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>136</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>удалить всё</string>
     </property>
    </widget>
    <widget class="QLabel" name="count_label">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>90</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>35</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: black;
color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="bg">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>-180</x>
       <y>-10</y>
       <width>951</width>
       <height>801</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="text">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 255, 255);</string>
     </property>
     <property name="text">
      <string>Добавте свой первый сон...</string>
     </property>
    </widget>
    <zorder>bg</zorder>
    <zorder>title</zorder>
    <zorder>button_what</zorder>
    <zorder>deleteAll_button</zorder>
    <zorder>count_label</zorder>
    <zorder>button_add</zorder>
    <zorder>text</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
